maven/bnd-export-plugin/src/main/java/aQute/bnd/maven/export/plugin/ExportMojo.java

```diff
@@ -0,0 +1,68 @@
+package aQute.bnd.maven.export.plugin;
+
+import static org.apache.maven.plugins.annotations.LifecyclePhase.PACKAGE;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+import org.apache.maven.plugin.AbstractMojo;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.apache.maven.plugins.annotations.ResolutionScope;
+
+import aQute.bnd.build.Container;
+import aQute.bnd.build.StandaloneRun;
+import aQute.bnd.osgi.Jar;
+import biz.aQute.resolve.ProjectResolver;
+
+@Mojo(name = "export", defaultPhase = PACKAGE, requiresDependencyResolution = ResolutionScope.COMPILE_PLUS_RUNTIME)
+public class ExportMojo extends AbstractMojo {
+
+	@Parameter(readonly = true, required = true)
+	private File bndrun;
+
+	@Parameter(defaultValue = "${project.build.directory}", readonly = true)
+	private File targetDir;
+
+	@Parameter(readonly = true, required = false)
+	private boolean resolve = true;
```

Comment:
Well the default is to resolve which may not be the best default. But we also need to gain support from the other project maintainers  here.